src/components/SubPage/EditingDesign.tsx

```diff
@@ -43,10 +43,24 @@ const EditingDesignSubpage: React.FC<EditingDesignSubpageProps> = ({
     }
 
     if (dnaDonorSequence) {
+        const sequences = Array.isArray(dnaDonorSequence)
+            ? dnaDonorSequence
+            : [dnaDonorSequence];
```

Comment:
this shouldn't really be necessary right? you've now converted everything to an array

Reply:
oh good catch! Will fix it now.